redhat/squeezeboxserver.spec

```diff
@@ -56,7 +56,13 @@ Vendor:		Lyrion Community
 
 Requires:	perl >= 5.10.0
 Recommends:     perl(IO::Socket::SSL)
-Obsoletes:	squeezeboxserver, squeezecenter, slimserver, SliMP3
+
+Provides:	logitechmediaserver = %{version}-%{release}
```

Comment:
Shouldn't this be `lyrionmusicserver`?

Reply:
No.  We want to tell RPM that if someone/something asks for `logitechmediaserver`, this package is a suitable substitute.

Reply:
"Someone" shouldn't be asking for it any more, as we're no longer allow to use that name. What could "something" be? Would the application still be known by the name of "lyrionmusicserver" if someone/something decided to install that?

Reply:
Let's say someone has packaged their LMS plugin as an RPM.  They'll currently have something like the following in their plugin's spec file:

```rpm-spec
Requires: logitechmediaserver > 8.1
```

Adding `Provides: logitechmediaserver` to our spec file allows that to continue to work until they get around to updating their spec to, say:

```rpm-spec
Requires: (logitechmediaserver > 8.1 or lyrionmusicserver)
```


Regarding what the application would be known as, I currently have:

```console
$ rpm -q logitechmediaserver
logitechmediaserver-8.5.0-1.noarch
$ rpm -q --whatprovides logitechmediaserver
logitechmediaserver-8.5.0-1.noarch
$ rpm -q lyrionmusicserver
package lyrionmusicserver is not installed
$ rpm -q --whatprovides lyrionmusicserver
no package provides lyrionmusicserver
```

Before this merge request, that becomes:

```console
$ rpm -q logitechmediaserver
package logitechmediaserver is not installed
$ rpm -q --whatprovides logitechmediaserver
no package provides logitechmediaserver
$ rpm -q lyrionmusicserver
lyrionmusicserver-9.0.0-0.1.1712141947.noarch
$ rpm -q --whatprovides lyrionmusicserver
lyrionmusicserver-9.0.0-0.1.1712141947.noarch
```

After this merge request, we eventually get to:

```console
$ rpm -q logitechmediaserver
package logitechmediaserver is not installed
$ rpm -q --whatprovides logitechmediaserver
lyrionmusicserver-9.0.0-1.noarch
$ rpm -q lyrionmusicserver
lyrionmusicserver-9.0.0-1.noarch
$ rpm -q --whatprovides lyrionmusicserver
lyrionmusicserver-9.0.0-1.noarch
```